platform-ajax/src/main/java/com/softicar/platform/ajax/event/AjaxDomEventService.java

```diff
@@ -4,9 +4,7 @@
 import com.softicar.platform.ajax.document.service.AbstractAjaxDocumentActionService;
 import com.softicar.platform.ajax.exceptions.AjaxHttpInternalServerError;
 import com.softicar.platform.ajax.request.IAjaxRequest;
-import com.softicar.platform.dom.event.IDomEvent;
-import com.softicar.platform.dom.event.IDomEventHandler;
-import com.softicar.platform.dom.exception.IDomExceptionDisplayElement;
+import com.softicar.platform.dom.event.DomEventHandlerNodeCaller;
 import com.softicar.platform.dom.node.IDomNode;
 
 public class AjaxDomEventService extends AbstractAjaxDocumentActionService {
```

Comment:
the extracted code only existed here in this class, but is actually required to be executed also for the `DomDocumentTestEngine`, so I extracted it to `DomEventHandlerNodeCaller`.